spec/builders/messages/facebook/message_builder_spec.rb

```diff
@@ -58,5 +58,88 @@
       expect(facebook_channel.inbox.reload.contacts.count).to eq(1)
       expect(contact.name).to eq(default_name)
     end
+
+    context 'when lock to single conversation' do
+      subject(:mocked_message_builder) { described_class.new(fb_message.new(contact_inbox.source_id, {}, '', '', ''), facebook_inbox).perform }
+
+      let!(:fb_message) { Struct.new(:sender_id, :attachments, :content, :identifier, :in_reply_to_external_id) }
+      let!(:account) { create(:account) }
+      let!(:facebook_inbox) { create(:inbox, channel: facebook_channel, account: account, greeting_enabled: false) }
```

Comment:
you can access the inbox through . `facebook_channel.inbox`  , creating a new inbox with the `facebook_channel` object will result in two inboxes in the system with same channel object, with isn't a pattern we expect.